src/simtools/schemas/model_parameters/flasher_photons.schema.yml

```diff
@@ -1,22 +1,23 @@
 %YAML 1.2
 ---
-title: Schema for laser_var_photons model parameter
+title: Schema for flasher_photons model parameter
```

Comment:
flasher bunch size and this one here seems to describe the same. Can we just use one?

Reply:
Ok then it would be good to specify what is meant as a bunch in the flasher_bunch_size.